### ISSUE TITLE
How to check if the sftp session is disconnected

### ISSUE THREAD

**Issue by Kannaj** [openssh-rust/openssh-sftp-client#116]
Hi,

Is there a way to check if an sftp session got disconnected? I am planning to maintain long running sftp connections to a remote server and would ideally like to reconnect in case of intermittent disconnections

```
openssh-sftp-client = { version = "0.13.6", features = ["openssh"] 
```

the underlying ssh session has a `.check()` method. but is there a way to access it from the sftp session?

Thanks

**Comment by NobodyXu** [comment]
I can add a new method that takes a closure for checking interval.

Can't expose the session directly because we need to hold on to call `Session::close`, so we also can't wrap it in an `Arc`

**Comment by NobodyXu** [comment]
cc @Kannaj Released openssh-sftp-client 0.14.2

**Comment by Kannaj** [comment]
@NobodyXu - wow , thank you so much for the quick fix! I'll test my flow with the new update.
